application/backend/src/main/resources/apps/typerefinery/components/widgets/columnticker/head-libs.html

```diff
@@ -1,2 +1,3 @@
-<script src="/apps/typerefinery/components/widgets/smallticker/script.js"></script>
+<script src="/apps/typerefinery/components/widgets/smallticker/clientlibs/behaviour.js"></script>
```

Comment:
Please take care of the imports.

Test before PR.

Reply:
Removed